.github/dependabot.yml

```diff
@@ -34,7 +35,8 @@ updates:
     open-pull-requests-limit: 20
```

Comment:
Bump this up.
We have in 10.0.x an average of 6.8 dependabot PRs per week.
Lets make 10.0.x have a limit above 30, lets use 50.

Raw numbers from the start of the year.
```
10.0.x - Daily updates
   
  is:pr created:>=2022-01-01 head:jetty-10.0.x 
     18 open
    264 closed
    (12 PRs per week)
  is:pr created:>=2022-01-01 base:jetty-10.0.x label:dependencies 
      4 open
    151 closed
   (6.8 PRs per week)
  is:pr created:>=2022-01-01 base:jetty-10.0.x label:dependencies is:merged 
    127 merged
   (5.7 PRs per week) 
  is:pr created:>=2022-01-01 base:jetty-10.0.x label:dependencies is:unmerged 
     28 un-merged
   (22% not merged)
```